operator/api/v1alpha1/common/common.go

```diff
@@ -45,3 +49,23 @@ type CheckType string
 
 const PreDeploymentCheckType CheckType = "pre"
 const PostDeploymentCheckType CheckType = "post"
+
+type KeptnMeters struct {
+	TaskCount          syncint64.Counter
+	TaskDuration       syncfloat64.Histogram
+	TaskActive         syncint64.UpDownCounter
+	DeploymentCount    syncint64.Counter
+	DeploymentDuration syncfloat64.Histogram
+	DeploymentActive   syncint64.UpDownCounter
+}
+
+const (
+	ApplicationName  attribute.Key = attribute.Key("keptn.deployment.app_name")
+	Workload         attribute.Key = attribute.Key("keptn.deployment.workload")
+	Version          attribute.Key = attribute.Key("keptn.deployment.version")
+	Namespace        attribute.Key = attribute.Key("keptn.deployment.namespace")
+	DeploymentStatus attribute.Key = attribute.Key("keptn.deployment.status")
+	TaskStatus       attribute.Key = attribute.Key("keptn.task.status")
+	TaskName         attribute.Key = attribute.Key("keptn.task.name")
+	TaskType         attribute.Key = attribute.Key("keptn.taks.type")
```

Comment:
I have second thouhgts on this. These attributes are specific of a given deployment since a task cannot be executed if they are not part of a deployment. 
I think it is better to put all of them behind the same grouping: `keptn.deployment.task.*`